test/cql-pytest/test_select_mutation_fragments.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+# Copyright 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# Tests concerning the SELECT MUTATION FRAGMENT statement, which allows dumping
+# the underlying mutation fragment data stream, for a table.
+
+
+import cassandra.protocol
+import nodetool
+import pytest
+import requests
+import util
+
+
+@pytest.fixture(scope="module")
+def test_table(cql, test_keyspace):
+    """ Prepares a table for the mutation dump tests to work with."""
+    with util.new_test_table(cql, test_keyspace, 'pk1 int, pk2 int, ck1 int, ck2 int, v text, s text static, PRIMARY KEY ((pk1, pk2), ck1, ck2)') as table:
+        yield table
+
+
+def test_smoke(cql, test_table):
```

Comment:
It seems that all tests in this file are Scylla-only because they test a feature that doesn't exist in Cassandra. Please either add a "scylla_only" to each test, or perhaps even better in this case, add a auto-use fixture which will add scylla_only to each and every test in this file. Something like (unchecked):
```
@pytest.fixture(scope="function", autouse=True)
def all_scylla_only(scylla_only):
    pass
```

The idea is that `test/cql-pytest/run-cassandra test_select_mutation_fragments.py` will skip all these tests.